telemetry/record.go

```diff
@@ -95,3 +106,27 @@ func (record *Record) ensureEncoded() {
 		record.Serializer.Logger().Error("record_RawBytes_blank")
 	}
 }
+
+func getPayloadWithVin(record *Record) ([]byte, error) {
+	if record.TxType == "alerts" {
```

Comment:
minor: can we use a `switch` to improve readability and ease of adding new records?